### DISCUSSION TITLE
Rust SDK

### DISCUSSION THREAD

**Discussion by JRazek** [deepgram/community#151] [General help] [answered]
I'm researching your Rust SDK. Project seems to be dead though. Is there any particular reason for that and is the development going to be continued any time soon?

**Comment by lukeocodes** [comment, marked as answer]
Hey @JRazek, our [Rust SDK](https://github.com/deepgram-devs/deepgram-rust-sdk) is currently community maintained. As far as I'm aware, it's currently working but missing some features. There is no short-term plan to make this an official SDK, but our long term goal is to adopt it. 

We welcome all contributions, though! So if you'd like to contribute any changes, we'd try to review them in a timely manner.
